modules/backend/layouts/auth.htm

```diff
@@ -10,15 +10,42 @@
         <link rel="icon" type="image/png" href="<?= e(Backend\Models\BrandSetting::getFavicon()) ?>">
         <title><?= e(trans('backend::lang.auth.title')) ?></title>
 
+        <script data-cfasync="false" src="<?= Backend::skinAsset('assets/js/vendor/jquery.min.js') ?>?v<?= $coreBuild ?>"></script>
+        <script data-cfasync="false" src="<?= Backend::skinAsset('assets/js/vendor/jquery-migrate.min.js') ?>?v<?= $coreBuild ?>"></script>
+
+        <link rel="preload" href="<?= Backend::skinAsset('assets/js/vendor/jquery.min.js') ?>?v<?= $coreBuild ?>" as="script">
+        <link rel="preload" href="<?= Backend::skinAsset('assets/js/vendor/jquery-migrate.min.js') ?>?v<?= $coreBuild ?>" as="script">
+
+        <!-- Inline Javascript to reduce lookups -->
+        <script>
+        // Load the Preload Assets	
+        $(document).ready(function(){
+            $(document.body).removeClass('preload')
+
+            $('form input[type=text], form input[type=password]').first().focus()
+        })
+
+        /* Only run on HTTPS connections
+         * Block off Front-end Service Worker from running in the Backend allowing security injections see GitHub #4384
+         */ 
+         if (location.protocol === 'https:') {
+             // Unregister all service workers before signing in to prevent cache issues see GitHub #4088
+             navigator.serviceWorker.getRegistrations().then(
+                 function(registrations) {
+                     for (let registration of registrations) {  
+                         registration.unregister();
+                     }
+                 });
+             }
+        </script>
```

Comment:
indentation on this block is still incorrect. Also, please don't include the jQuery stuff hardcoded, just move this script tag to after the jQuery resources would be loaded normally

Reply:
@LukeTowers Ok I have done what you have suggested, please let me know if I did it wrong.